about/our-team.php

```diff
@@ -220,45 +262,45 @@
                         </div>
                     </div>
                 </div>
+
                 <!-- Event Coordinator -->

```

Comment:
Swap this

Reply:
swap what?

Reply:
I think Nick is referring to the fact that Jesse is Aerospace Division Lead, not Event Coordinator :)